v22.1/ui-statements-page.md

```diff
@@ -7,12 +7,31 @@ docs_area: reference.db_console
 
 {% include {{ page.version.version }}/ui/admin-access.md %}
 
-The **Statements** page helps you:
+The **Statements** page  of the DB Console helps you:
 
 - Identify frequently executed or high latency [SQL statements](sql-statements.html).
 - View SQL statement [details](#statement-details-page).
 - Download SQL statement [diagnostics](#diagnostics) for troubleshooting.
 
 To view this page, click **SQL Activity** in the left-hand navigation of the DB Console. The **Statements** tab is selected.
 
+{{site.data.alerts.callout_success}}
+If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+{{site.data.alerts.end}}
+
+## Search and filter
+
+By default, the Statements page shows SQL statements from all applications and databases running on the cluster.
+
+### Search statements and statement plans
+
+To search using the search field, type a string over `Search Statements` and press `Enter`. The search will look for matches in:
+
+- Statement fingerprints.
+- <span class="version-tag">New in v22.1:</span> [Statement plans](#explain-plans). For example, suppose you want to identify statements using a specific index. You can search for a specific index and if the index appears in the statement plan text the search will return that statement fingerprint.
```

Comment:
@maryliag, do you know if Gerardo's changes for searching by index is supported against all plans? Since we capture all plans now in 22.1, did this break/impact Gerardos feature to search by index at all?

Reply:
is just looking at the sampled plan. It would be to expensive to decode every single plan to do the search on top of it

Reply:
Every time we execute a statement, it uses a plan, we sample a few of them to get the most recent stats from it.

@kevin-v-ngo I don't know how we can describe this, because is not straight forward to tell the user which one was sample, also we might stop doing the sampling part, since we no longer the sample plan anywhere. Do you think is worth removing this part of the docs?

Reply:
Yes, we should remove it since the sampled plan is no longer being used anywhere and we are tracking to remove it in the future since it doesn't represent the actual plan and could be a false signal.

I do wonder if there is a way we can perform the search so that it's not expensive. This was a great feature. Perhaps we can file a tracking improvement here and revisit this especially with future improvements with the obs service.

Reply:
@kevin-v-ngo I'm not sure what you are referring to by "remove it". Do you mean remove the entire bullet that says you can search plans?

Reply:
FYI - https://github.com/cockroachdb/cockroach/issues/81485